modules/get-started/pages/licensing/check-status/rpk.adoc

```diff
@@ -0,0 +1,82 @@
+= Check License Status and Feature Usage with `rpk`
+:description: pass:q[Learn how to monitor the status of an Enterprise Edition license and enterprise feature usage with `rpk`.]
+
+To check the status of your license, use the `rpk cluster license info` command. This command provides a detailed overview of your Redpanda license, including its status, expiration, and enterprise features currently in use.
+
+. Make sure that xref:get-started:rpk-install.adoc[`rpk` is installed] and configured to connect to your cluster's Admin API endpoint.
+
+. Get the details about your cluster's license:
++
+```bash
+rpk cluster license info
+```
++
+The command displays the license information in a user-friendly format. The output depends on the license status. It can include the following:
++
+include::get-started:partial$licensing/license-status.adoc[]
+
+If the license is within 30 days of expiration, a warning is logged. `rpk` displays warnings when you execute `rpk` commands that use the Admin API in the following scenarios:
+
+- *License violation*: When enterprise features are enabled without a valid license.
+
+- *Trial expiration*: When enterprise features are enabled and a trial license expires in less than 15 days.
+
+- *Enterprise expiration*: When enterprise features are enabled and an enterprise license is expired.
+
+== Examples
+
+This section provides examples of what `rpk` reports depending on the license status.
+
+Valid license:
+
+[.no-copy]
+----
+LICENSE INFORMATION
+===================
+License status:              valid
+License violation:           false
+Enterprise features in use:  [partition_auto_balancing_continuous]
+Organization:                Devex
+Type:                        enterprise
+Expires:                     Oct 11 2025
+----
+
+Without a valid license:
+
+[.no-copy]
+----
+LICENSE INFORMATION
+===================
+License status:              not_present
+License violation:           false
+Enterprise features in use:  [partition_auto_balancing_continuous]
+----
+
+Expired license:
+
+[.no-copy]
+----
+WARNING: The following enterprise features are being used in your Redpanda cluster: [partition_auto_balancing_continuous]. These features require a license.
+LICENSE INFORMATION
+===================
+License status:              expired
+License violation:           true
+Enterprise features in use:  [partition_auto_balancing_continuous]
+----
+
+== Change the output format
+
+Different output formats can be useful depending on your scenario. For example, if you are writing scripts or automating license monitoring, you may prefer the JSON format, as it's easily parsed by tools like `jq` or integrated into monitoring systems.
+
+To get the license information in another format, use the `--format` flag:
+
+```bash
+rpk cluster license info --format <format>
+```
+
+Replace `<format>` with one of the available formats. See xref:reference:rpk/rpk-cluster/rpk-cluster-license-info.adoc[].
```

Comment:
we could list the formats here to avoid user hoping into the next page. If integrity between this page and the rpk page are a concern, we write it to be broad.

Something like 
```suggestion
Replace `<format>` with one of the available formats, such as `json`, or `yaml`. For more formats, see xref:reference:rpk/rpk-cluster/rpk-cluster-license-info.adoc[].
```